src/main/java/att/grappa/AwtExceptionDisplay.java

```diff
@@ -0,0 +1,203 @@
+/*
+ *  This software may only be used by you under license from AT&T Corp.
+ *  ("AT&T").  A copy of AT&T's Source Code Agreement is available at
+ *  AT&T's Internet website having the URL:
+ *  <http://www.research.att.com/sw/tools/graphviz/license/source.html>
+ *  If you received this software without first entering into a license
+ *  with AT&T, you have an infringing copy of this software and cannot use
+ *  it without violating AT&T's intellectual property rights.
+ */
+package att.grappa;
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Frame;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import java.awt.Panel;
+import java.awt.TextArea;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+/**
+ * A class for displaying exception information in a pop-up frame. As a convenience, an instance exists as a static
+ * member of the <code>Grappa</code> class.
+ *
+ * @see Grappa#displayException(java.lang.Exception)
+ * @see Grappa#displayException(java.lang.Exception,java.lang.String)
+ * @version $Id$
+ * @author <a href="mailto:john@research.att.com">John Mocenigo</a>, <a href="http://www.research.att.com">Research @
+ *         AT&T Labs</a>
+ */
+public class AwtExceptionDisplay
+    implements ExceptionDisplay
+{
+    private String title = null;
+
+    Exception exception = null;
+
+    Display display = null;
+
+    /**
+     * Creates an instance of the class for displaying exceptions.
+     *
+     * @param title the title for the pop-up frame
+     */
+    public AwtExceptionDisplay(String title)
+    {
+        this.title = title;
+    }
+
+    /**
+     * Pops up the frame and displays information on the supplied exception. Initially, a text area displays the message
+     * associated with the exception. By pressing a button, an end-user can view a stack trace as well.
+     *
+     * @param ex the exception about which informtaion is to be displayed.
```

Comment:
You didn't fix the `informtaion` typo here, but you did in the original fine. Can you fix it here too?

Reply:
I wanted to make the least modification possible, but of course I can do this.